src/templates/docs/vertical-align/index.html

```diff
@@ -45,20 +45,22 @@ <h4 class="f4 mv0 fw6 dib mr4"><%= name %></h4>
       <div class="ph3 ph5-ns pt4 pb5">
         <h2 class="f4 bold">Examples</h2>
         <h3 class="f5 book sans-serif pt4 caps">Inline Alignment</h3>
-        <p class="f4 measure">
+        <code class="f6 db gray">.v-sub</code>
+        <p class="f4 measure mt2 mb5">
           Lorem ipsum dolor sit amet, consetetur sadipscing
           <span class="v-sub bg-lightest-red">Sub aligned text (v-sub)</span>
           elitr, sed diam nonumy eirmod
+        </p>
+        <code class="f6 db gray">.v-sup</code>
+        <p class="f4 measure mt2">
           tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
           <span class="v-sup bg-lightest-red">Super aligned text (v-sup)</span>
           vero eos et accusam et justo duo dolores et ea rebum.
         </p>
         <h3 class="f5 book sans-serif pt4 caps">Table Alignment</h3>
-        <table border="0" cellspacing="8" cellpadding="8">
+        <code class="f6 db gray">.v-mid</code>
+        <table border="0" cellspacing="8" cellpadding="8" class="mb5 mt2">
           <tr>
-            <td>
```

Comment:
Removed td with class name in and instead added class name usage above as in other pages